specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2022-08-01/virtualMachineScaleSet.json

```diff
@@ -3925,6 +3931,13 @@
             "lun"
           ],
           "description": "Specifies the parameters that are used to add data disks to the virtual machines in the scale set. <br><br> For more information about disks, see [About disks and VHDs for Azure virtual machines](https://docs.microsoft.com/azure/virtual-machines/managed-disks-overview)."
+        },
+        "diskControllerType": {
+          "type": "string",
+          "items": {
```

Comment:
I think the keyword "items" is for arrays https://swagger.io/docs/specification/data-models/data-types/ (@grizzlytheodore can you confirm or correct my understanding?) you only need to do $ref and description similar to osDisk

Maybe that is why some of the validations are failing?

Reply:
Addressed this.It was an issue with referencing